recommenders/__init__.py

```diff
@@ -2,7 +2,7 @@
 # Licensed under the MIT License.
 
 __title__ = "Microsoft Recommenders"
-__version__ = "0.7.0"
+__version__ = "0.8.0"
```

Comment:
I think this is a big update with spark 3 and tf 2 support. One can even argue for a bump to `1.0.0` when our dependencies have major bumps (spark 2->3, tf 1->2). I think these many changes, there's a lot of risk of incompatible changes that is not backward compatible.

From the language of semantics versioning, I think we could consider a bump in the major version to `1.0.0`:
> 
1. MAJOR version when you make incompatible API changes,
2. MINOR version when you add functionality in a backwards compatible manner, and
3. PATCH version when you make backwards compatible bug fixes.

Reply:
makes sense

Reply:
Just made the change to 1.0.0 🍾

Reply:
hey guys, let's consider the version in the next reco meeting, I think going to 1.0.0 is a big deal and it would be good to get the thoughts of the team here